lib/husky_musher/app.py

```diff
@@ -64,10 +64,10 @@ def main():
     # Get NetID and other attributes from Shibboleth data
     if DEVELOPMENT_MODE:
         remote_user = os.environ.get("REMOTE_USER")
-        user_info = extract_user_info(os.environ)
```

Comment:
If its no longer used, can we remove the methods from `utils/shibboleth.py`? Or, if we prefer to keep the abstraction, perhaps edit `extract_user_info()` to only set netid?

Reply:
I started out by modifying `extract_user_info()`, but it ended up only being one line of code.  I also thought it may be worth keeping the methods in `utils/shibboleth.py` in case they change their mind about dropping these fields, but they're not currently being used.

Reply:
I decided to just drop `shibboleth.py`, we can always bring it back if we need to.

Reply:
Yeah it would certainly be pretty simple if we wanted to use it for just netid. Sounds good